tests/mechanic/provisioner_test.py

```diff
@@ -40,11 +43,11 @@ def null_apply_config(source_root_path, target_root_path, config_vars):
         team.Car(
             names="unit-test-car",
             root_path=None,
-            config_paths=["~/.rally/benchmarks/teams/default/my-car"],
+            config_paths=[f"{HOME_DIR}/.rally/benchmarks/teams/default/my-car"],
             variables={"heap": "4g"}),
             java_home="/usr/local/javas/java8",
             node_name="rally-node-0",
-            node_root_dir="~/.rally/benchmarks/races/unittest",
+            node_root_dir=f"{HOME_DIR}/.rally/benchmarks/races/unittest",
```

Comment:
Python f-Strings are really great, but unfortunately we perform integration tests across 3.5, 3.6 and 3.7 and f-Strings became available only starting with Python 3.6.

To catch these it's recommended to run `make it` to run the integration test-suite.